auth-server/src/services/email.ts

```diff
@@ -38,9 +38,12 @@ export const sendVerificationEmail = (user: User, token: EmailVerificationToken)
 		html: text
 	}
 
-	console.log(`Sending verification email to ${user.email}`)
-
-	sgMail.send(msg);
+	try {
+		sgMail.send(msg);
```

Comment:
await here other wise you will get UnhandledPromiseRejectionWarning

Reply:
Better use .catch(error=> ...) otherwise you will have to convert function to async. we want this request to be running in background